packages/api/helper/mailer.js

```diff
@@ -36,13 +36,25 @@ const email = new emailTemplate({
   preview: false,
 });
 
+function validateEmail(email) {
+  const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
It's generally a bad idea to validate emails with regex because the conditions for what qualifies as a valid email are quite complex. See more here if you're curious: https://stackoverflow.com/questions/48055431/can-it-cause-harm-to-validate-email-addresses-with-a-regex In this case I don't think we need to validate the email at all since this is just an internal mechanism that won't be user-facing.

Reply:
Also removed validateEmail function.